common/pointer.libevm.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2025 the libevm authors.
+//
+// The libevm additions to go-ethereum are free software: you can redistribute
+// them and/or modify them under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation, either version 3 of the License,
+// or (at your option) any later version.
+//
+// The libevm additions are distributed in the hope that they will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser
+// General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see
+// <http://www.gnu.org/licenses/>.
+
+package common
+
+// PointerTo is a convenience wrapper for creating a pointer to a value.
+func PointerTo[T any](x T) *T { return &x }
```

Comment:
I'd lean towards having a local unexported function, and just copy paste. This is just a one-liner, it's worth removing a package dependency etc. and creating a common package opens the gates for... anything and everything 😄

Reply:
Usually I'd absolutely agree and would be against an attic package where you put all your stuff that has no other place to go. The reason I placed it here is because the `common` package is so ubiquitous in `geth` and I've lost count of the number of times that `PointerTo` would have been useful for this precise case of `common.Address` construction.